stream/stream.c

```diff
@@ -569,7 +569,10 @@ bool stream_seek(stream_t *s, int64_t pos)
     }
 
     if (s->mode == STREAM_WRITE)
-        return s->seekable && s->seek(s, pos);
+        if (s->seekable)
+            return s->seek(s, pos);
+        else
+            return s->seekable;
```

Comment:
== `return false;`

Reply:
Hi @sfan5, do you mean (s->seekable == true) and return false in the else branch?

Reply:
I mean that the else branch is equivalent to `return false;` so you should just make the explicit.